lib/themes/dosomething/paraneue_dosomething/templates/system/partials/footer.tpl.php

```diff
@@ -9,36 +9,37 @@
 ?>
 
 <footer class="footer<?php if ($is_affiliate): ?> -affiliate <?php endif; ?>">
+  <div class="footer__columns">
```

Comment:
could this just be a `.wrapper` class and then the `.footer__column` sits inside that instead of plural/singular thing?